src/Components/Signup.js

```diff
@@ -67,6 +77,29 @@ const Signup = () => {
         }
     }
 
+    const formValidator = () => {
+        if (!isEmailError && !isPassword1Error && !isPassword2Error && !isUsernameError) {
+            setIsFormValid(true);
+        }
+    }
+
+    const formSubmitHandler = (event) => {
+        event.preventDefault();
+        formValidator();
+        if (isFormValid) {
+            Axios.post("http://localhost:3000/signup", { username: enteredUsername, password: enteredPassword2, email: enteredEmail })
+                .then()
+                .catch(err => {
+                    if (err.response.status == 400) {
+                        alert("Username is already taken.");
```

Comment:
No alerts. Just leave it as a console.error and we can figure out the details later